core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -6,7 +6,7 @@
     <i class="material-icons oppia-navbar-close-icon" ng-if="$ctrl.isSidebarShown()" >&#10005;</i>
   </a>
 
-  <a class="oppia-navbar-brand-name oppia-transition-200 protractor-test-oppia-main-logo d-none d-sm-block" href="/"
+  <a class="oppia-navbar-brand-name oppia-transition-200 protractor-test-oppia-main-logo d-none d-sm-block" href="/splash"
```

Comment:
I don't understand why this PR is changing certain things to /splash if we are deprecating /splash.

Reply:
PTAL in the thread [above](https://github.com/oppia/oppia/pull/9109#discussion_r410668909).  
This would redirect the user based on the login status and now, instead of `/`, the `/splash` is being used which explains this change.